<launch>
  <node pkg="firefly_mapping" type="onboard_mapping_test" name="onboard_mapping_test" output="screen"/>
  <node pkg="firefly_perception" type="firefly_perception_test" name="firefly_perception_test" output="screen"/>
  <node pkg="tf" type="static_transform_publisher" name="base_to_thermal_camera_link" args="-0.01568547 -0.041098 -0.27658746 1.5313006 0.0074513 -3.1380038 base_link thermal/camera_link 100"/>
  <node pkg="firefly_bringup" type="gps_to_local_enu.py" name="gps_to_local_enu"/>
</launch>